core/client/fs/src/main/java/alluxio/client/file/cache/store/MemoryPageStore.java

```diff
@@ -94,4 +107,52 @@ public void close() {
   public void reset() {
     mPageStoreMap.clear();
   }
+
+  private static class MemPage {
+    private final byte[] mPage;
+    private int mPageLength;
+
+    public MemPage(byte[] page, int pageLength) {
+      mPage = page;
+      mPageLength = pageLength;
+    }
+
+    public byte[] getPage() {
+      return mPage;
+    }
+
+    public int getPageLength() {
+      return mPageLength;
+    }
+
+    public void setPageLength(int pageLength) {
+      mPageLength = pageLength;
+    }
+  }
+
+  private static class PagePool {
+    private final int mPageSize;
+    private final LinkedList<MemPage> mPool = new LinkedList<>();
```

Comment:
How to completely free `mPool`? It seems that this can only be release by JVM GC.

Reply:
> How to completely free `mPool`? It seems that this can only be released by JVM GC.

good call, will try to release mPool in close,  I will fix it in anther pr